lib/src/types/relation.rs

```diff
@@ -35,6 +35,13 @@ impl BoltRelation {
     }
 }
 
+
+impl BoltRelation {
```

Comment:
There should be another impl for BoltUnboundedRelation.

Reply:
Good catch, thanks. Added in last commit.